xclim/indices/_conversion.py

```diff
@@ -895,6 +903,59 @@ def rain_approximation(
     return prra
 
 
+def snow_depth_approximation(
+    snw: xr.DataArray,
+    snr: xr.DataArray | None = None,
+) -> xr.DataArray:
+    """Approximation of snow density from snow water.
+
+    Parameters
+    ----------
+    snw : xr.DataArray
+        Snow water (kg/m^2).
+        If snow thickness (`swe`: "m") is provided, will be converted to `snw` before calculating.
+    snr : xr.DataArray, optional
+        Snow density (kg/m^3).
+
+    Returns
+    -------
+    xr.DataArray
```

Comment:
```suggestion
    xr.DataArray, [m]
        Snow depth.
```
Second line parsed as `long_name`in a potential indicator.